lib/terraorg/model/org.rb

```diff
@@ -134,10 +134,12 @@ def validate!(strict: true)
     end
 
     # Validate that any associate is a member of some squad
-    associates_but_not_members = Set.new(all_associates.map(&:id)) - Set.new(all_members.map(&:id)) - exceptions
-    if !associates_but_not_members.empty?
-      $stderr.puts "ERROR: #{associates_but_not_members} are associates of squads but not members of any squad"
-      failure = true
+    if !allow_orhpaned_associates
+      associates_but_not_members = Set.new(all_associates.map(&:id)) - Set.new(all_members.map(&:id)) - exceptions
+      if !associates_but_not_members.empty?
+        $stderr.puts "ERROR: #{associates_but_not_members.to_a().join()} are associates of squads but not members of any squad"
```

Comment:
```suggestion
        $stderr.puts "ERROR: #{associates_but_not_members.to_a} are associates of squads but not members of any squad"
```

take out the join, it won't make sense if there are multiple members:

```ruby
irb(main):001:0> ['a','b'].join
=> "ab"
```

The regular `to_a` formatting should work ok:

```ruby
irb(main):004:0> a=['a','b']
irb(main):005:0> puts("bla bla #{a}")
bla bla ["a", "b"]
```

Also, it is idiomatic to omit () for argumentless functions in Ruby.